Packs/SysAid/Integrations/SysAid/README.md

```diff
@@ -19553,6 +19553,66 @@ There is no context output for this command.
 
 >File uploaded to Service Record 25 successfully.
 
+### sysaid-service-record-get-file
+
+***
+Download an attachment to a service record.
+
+#### Base Command
+
+`sysaid-service-record-get-file`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | The service record ID. | Required | 
+| file_id | The ID of the file to download. | Required | 
+| file_name | The full name with extension of the file to be downloaded. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| File.Size | Integer | The size of the file. | 
+| File.SHA1 | String | The SHA1 of the file. | 
+| File.SHA256 | String | The SHA256 of the file. | 
+| File.SHA512 | String | The SHA512 of the file. |
+| File.Name | String | The full name with extension of the file | 
+| File.SSDeep | String | The SSDeep of the file | 
+| File.EntryID | String | The entryId of the file | 
+| File.Info | String | The info of the file | 
+| File.Type | String | The type of the file | 
+| File.MD5 | String | The MD5 of the file | 
+| File.Extension | String | The extension of the file | 
+
+There is no context output for this command.
```

Comment:
It's a little bit confused if there is output for this command (lines 19573 and 19589)

Reply:
The output of the "fileResult" comes out exactly that way, I have attached the example screenshot of a file downloaded with the command "!sysaid-service-record-get-file file_id=FILEID file_name=FILENAME id=IDTICKET"

![Screenshot 2023-11-02 093719](https://github.com/demisto/content/assets/147415442/067ddf45-823f-42f6-946d-96cfa25425b8)